src/schema/v2/artwork/index.ts

```diff
@@ -82,6 +82,21 @@ export const ArtworkImportSourceEnum = new GraphQLEnumType({
   values: IMPORT_SOURCES,
 })
 
+const MarketPriceInsightsType = new GraphQLObjectType<any, ResolverContext>({
```

Comment:
Is this actually working? There is already a type with this name in the schema which comes from Vortex.

Reply:
I had to add this type to be able to query for the insights within myCollectionConnection